nexus/tests/integration_tests/volume_management.rs

```diff
@@ -371,7 +371,7 @@ async fn test_multiple_snapshots(cptestctx: &ControlPlaneTestContext) {
     .unwrap();
 
     // Create a disk from this image
-    let disk_size = ByteCount::try_from(2u64 * 1024 * 1024 * 1024).unwrap();
+    let disk_size = ByteCount::try_from(1u64 * 1024 * 1024 * 1024).unwrap();
```

Comment:
FYI, there's `ByteCount::from_gibibytes_u32()` (which also cannot fail).  That and the related functions would eliminate a lot of unwraps here.  (not important)

Reply:
:+1: done in d2d3aefb